manim/scene/scene_file_writer.py

```diff
@@ -42,8 +42,9 @@ class SceneFileWriter(object):
 
     """
 
-    def __init__(self, scene, **kwargs):
+    def __init__(self, video_quality_config, scene, **kwargs):
```

Comment:
We should avoid importing any more configuration while working towards passing the currently imported correctly, because importing isn't the proper way to pass configuration. For example, there's no reason for the `SceneFileWriter` to import its own config rather than storing it.

Reply:
I get it, and I tend to agree with this but then it would imply to refactor the whole config system, and not only video_quality_config. For the moment, just for consistency sake, I think this should wait for another PR. 

cc @leotrs

Reply:
We do have to refactor the config system, but there's no benefit in continuing to pass the config incorrectly in the meantime. There's a decent chance we'll end up migrating from importing the config gradually anyway.